FindKqueue.cmake

```diff
@@ -140,12 +129,11 @@ if ( NOT HAVE_KQUEUE )
     endif ()
 
     add_library(libkqueue_a STATIC IMPORTED)
-    set_property(TARGET libkqueue_a PROPERTY IMPORTED_LOCATION
-      ${kqueue_build}/libkqueue${static_ext})
+    set_property(TARGET libkqueue_a PROPERTY IMPORTED_LOCATION ${kqueue_static})
     add_dependencies(libkqueue_a project_kqueue)
 
     set(HAVE_KQUEUE true)
-    set(LIBKQUEUE_LIBRARIES libkqueue_a CACHE STRING "libkqueue libs" FORCE)
+    set(LIBKQUEUE_LIBRARIES ${kqueue_static} CACHE STRING "libkqueue libs" FORCE)
```

Comment:
Curious if that's intentional or what the change accomplishes.  The former implies we end up depending on the name of the library target and the later implies we end up depending on an absolute path to the file?  Guess the later is fine if CMake dependency resolution just ends up associating the file path with the target anyway, but then the former does make that more explicit and logical to me.

Reply:
Went back to the old style, since it appears to work fine. It wasn't working originally so I must have fixed it with some other changes and never reverted that one.